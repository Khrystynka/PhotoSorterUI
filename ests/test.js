import React from "react";
import renderer from "react-test-renderer";
import Card from "../components/UI/Card";
import AllDocs from "../screens/uploads/AllDocumentsScreen";

test("renders correctly", () => {
	const tree = renderer
		.create(<Card style={{ borderColor: "red" }} />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

test("renders correctly", () => {
	const tree = renderer.create(<AllDocs />).toJSON();
	expect(tree).toMatchSnapshot();
});
