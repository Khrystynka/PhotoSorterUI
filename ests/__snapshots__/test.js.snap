// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "borderColor": "red",
      "borderRadius": 10,
      "elevation": 5,
      "height": "100%",
      "shadowColor": "black",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.3,
      "shadowRadius": 8,
      "width": "90%",
    }
  }
/>
`;
